test/service_failure_error_test.rb

```diff
@@ -1,30 +1,23 @@
 require 'test_helper'
 
-describe Circuitbox::ServiceFailureError do
+class ServiceFailureErrorTest < Minitest::Test
   class SomeOtherError < StandardError; end;
 
   attr_reader :error
 
-  before do
-    begin
-      raise SomeOtherError, "some other error"
-    rescue => ex
-      @error = ex
-    end
+  def setup
+    raise SomeOtherError, "some other error"
+  rescue => ex
+    @error = ex
```

Comment:
Well there is a different way but I find it awkward as probably less people will understand it...

```ruby
SomeError.new("message", caller)
```